cmake/pkg/postinst

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -e
+
+USER_NAME=clio
+GROUP_NAME=clio
+CLIO_PREFIX="/opt/clio"
+CLIO_BIN="$CLIO_PREFIX/bin/clio_server"
+CLIO_CONFIG="$CLIO_PREFIX/etc/config.json"
+
+case "$1" in
+    configure)
+        if ! id -u "$USER_NAME" >/dev/null 2>&1; then
+            adduser --system --quiet \
+                --home /nonexistent --no-create-home \
```

Comment:
Why do you need `--home /nonexistent` with `--no-create-home`?
This looks like a debug or some hack